packages/lisk-p2p/src/peer.ts

```diff
@@ -293,10 +323,20 @@ export class Peer extends EventEmitter {
 		if (!this._outboundSocket) {
 			this._outboundSocket = this._createOutboundSocket();
 		}
-		this._outboundSocket.emit(REMOTE_EVENT_MESSAGE, {
-			event: packet.event,
-			data: packet.data,
-		});
+		// TODO later: Legacy events will no longer be required after migrating to the LIP protocol version.
+		if (
+			packet.event === 'postBlock' || 
+			packet.event === 'postTransactions' || 
+			packet.event === 'postSignatures'
```

Comment:
maybe below, because lint is failing here
```
const legacyEvents = ['postBlock', 'postTransactions', 'postSignatures'];
if (legacyEvents.includes(packet.event))
```